solana/README.md

```diff
@@ -103,5 +106,42 @@ In order to run them, the Solana programs must be built and their IDL made avail
 To ensure the SDK has the generated IDL, run the tests with the make command:
 
 ```sh
-make anchor-test
-```
\ No newline at end of file
+make test
+```
+
+### Troubleshooting
+
+<details>
+<summary><code>make: *** No rule to make target `test'.  Stop.</code></summary>
+
+- Ensure `Makefile` has target `test`
+</details>
+
+<details>
+<summary><code>tsx: command not found</code></summary>
+
+- Screenshot:
+  <img src="images/tsx-command-not-found.png" alt="tsx command not found screenshot">
+- Update `Makefile` ([line #29](https://github.com/wormhole-foundation/example-native-token-transfers/blob/main/solana/Makefile#L29)) from:
+
+  ```sh
+  tsx scripts/regenerateIdl.ts $$jsonfile > $$tsfile; \
```

Comment:
wouldn't it make sense to just put into the prerequisites section how to install this binary?
encouraging local modifications sounds a bit dubious

Reply:
Yeah, my idea for the troubleshooting section was that it was specific to testing but I do see the merit in moving it to the prerequisites section as well.